.github/workflows/pages.yml

```diff
@@ -30,25 +30,33 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      # use doxygen to build the documentation for the C++ part
       - name: Doxygen action for libimgdoc2
         uses: mattnotmitt/doxygen-action@v1.9.5
         with:
           doxyfile-path: "./Doxyfile" # Path to Doxyfile
           working-directory: "libimgdoc2" # Working directory
-      - name: Doxygen action for ImgDoc2Net
-        uses: mattnotmitt/doxygen-action@v1.9.5
-        with:
-          doxyfile-path: "./Doxyfile" # Path to Doxyfile
-          working-directory: "dotnet/imgdoc2Net" # Working directory
+      # ...and docfx to build the documentation for the .NET part
+      # (main motivation for using docfx was that doxygen does not understand latest C# language features)
+      - name: DocFx action for ImgDoc2Net
+        run: |
+          dotnet tool update -g docfx
+          cd dotnet/docfx
+          docfx
+          # currently, we are building the documentation in-tree, and we manually are
+          # deleting the files from the repo here
+          rm docfx.json
+          rm toc.yml
+          rm index.md
       - name: Prepare doxygen-generated documentation for upload
         run: |
           mkdir ${{github.workspace}}/documentation
           mkdir ${{github.workspace}}/documentation/libimgdoc2
           mkdir ${{github.workspace}}/documentation/imgdoc2net
           cp -r ${{github.workspace}}/libimgdoc2/doxygen/html/* ${{github.workspace}}/documentation/libimgdoc2
-          cp -r ${{github.workspace}}/dotnet/imgdoc2Net/doxygen/html/* ${{github.workspace}}/documentation/imgdoc2net
-          echo "<meta http-equiv=\"Refresh\" content=\"0; url='libimgdoc2/'\" />" >> ${{github.workspace}}/documentation/index.html
-
+          cp -r ${{github.workspace}}/dotnet/docfx/* ${{github.workspace}}/documentation/imgdoc2net
+          # add a "top-level-HTML-file" redirecting to the actual documentation
+          echo "<meta http-equiv=\"Refresh\" content=\"0; url='imgdoc2net/'\" />" >> ${{github.workspace}}/documentation/index.html
```

Comment:
Just to reconfirm: So it is now intended to have the .NET API docs directly referenced from the GitHub landing page instead of the C++ API docs? Totally fine with me - I like the docfx docs much more since they even redirect back to the sources when clicking on "View source" :)

Reply:
Yes, that's the idea, and I put a link to the C++-part on the .NET-documentation start-page. I guess that's good enough for now. At some point in the future, I might reconsider/reorganize things, but for the time being I am fine with it as is. Yeah, it is a bit a pity that we now have two systems at work here (doxygen and docfx), but - those are just the go-to specialist in their respective domain.